core/src/main/java/com/amazon/randomcutforest/anomalydetection/AbstractAttributionVisitor.java

```diff
@@ -116,7 +116,7 @@ public void accept(Node node, int depthOfNode) {
         if (hitDuplicates || ignoreLeaf) {
             // use the sibling bounding box to represent counterfactual "what if point & the candidate near neighbor
             // had not been inserted in the tree"
-            Node sibling = isLeftOf(pointToScore, node) ? node.getRightChild() : node.getLeftChild();
+            Node sibling = Node.isLeftOf(pointToScore, node) ? node.getRightChild() : node.getLeftChild();
```

Comment:
Check to see if this is necessary, might have been added by intellij while moving files around.

Reply:
For the `isLeftOf` method, I think I actually prefer to keep the class name as part of the method call.